### PR TITLE
remove kubernetes-anywhere from this repository

### PR DESCRIPTION
**Pull request by neolit123**
**What this PR does / why we need it**:

The project has been deprecated [since Dec 2018](https://github.com/kubernetes/kubernetes-anywhere/commit/494d171d0b674).

Cleanup in:
- /test/e2e/framework/
- /cluster/

xref
https://github.com/kubernetes/test-infra/pull/17313
https://github.com/kubernetes/org/issues/1812
https://github.com/kubernetes/kubernetes/issues/78995

**Which issue(s) this PR fixes**:

NONE

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

/kind cleanup
/priority backlog

### CODE REVIEW

File: test/e2e/framework/test_context.go [hunk 1]
```diff
@@ -153,9 +153,6 @@ type TestContextType struct {
 	// Node e2e specific test context
 	NodeTestContextType
 
-	// Indicates what path the kubernetes-anywhere is installed on
-	KubernetesAnywherePath string
```
Comment (neolit123) [inline test/e2e/framework/test_context.go:157]: TestContextType does not have publicly exposed support guarantees, yet.

Comment (BenTheElder) [reply test/e2e/framework/test_context.go:157]: @alejandrox1 😬

**Comment by BenTheElder** [comment]
/retest
/lgtm

FYI @amwat re: kubetest

**Review by spiffxp — APPROVED** [review]
/approve
/lgtm

**Comment by neolit123** [comment]
> tide Pending — Not mergeable. Retesting: pull-kubernetes-kubemark-e2e-gce-big pull-kubernetes-verify


i think the status is stuck.

/test pull-kubernetes-kubemark-e2e-gce-big
/test pull-kubernetes-verify

**Pull request merged**